### PR TITLE
Add Mir engage page

### PR DESCRIPTION
**Pull request by solazio**
## Done

- Add Mir engage page

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [/engage/mir-for-iot](http://0.0.0.0:8001/engage/mir-for-iot)
- Run through the following [QA steps](https://canonical-web-and-design.github.io/practices/workflow/qa-steps.html)
- Compare to [brief](https://docs.google.com/document/d/11E9O_mVN_U63ZLsZOvwsdXAyNtpRv0aF8_VWBDfrnRc/edit#)
- Compare to [design](https://github.com/canonical-web-and-design/web-squad/issues/1181)
- Test it on [https://socialdebug.com/](https://socialdebug.com/) and [https://cards-dev.twitter.com/validator](https://cards-dev.twitter.com/validator)

## Issue / Card

Fixes https://github.com/canonical-web-and-design/web-squad/issues/1182

### CODE REVIEW

**Comment by solazio** [comment]
The pdf and marketo information is missing, hence QA -1

File: templates/engage/mir-for-iot.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+{% block content %}
+
+{% include "engage/shared/_header.html" with title="Build smart display devices with Mir: fast to production, secure, open-source" url="#the-form" cta="Download the whitepaper" class="p-strip--dark p-takeover--mir"%}
+
+<section class="p-strip is-shallow is-bordered">
+  <div class="row">
+    <div class="col-7">
+      <p>
+        Industrial robots, home appliances, advertising screens, office information boards...devices of every type around us are getting connected. As they do, their screens turn from single purpose displays to reconfigurable, multi-purpose smart display. As the amount of code required to build these displays, the production time and maintenance burden have increased, this has prompted device manufacturers to reconsider how they can build smart display devices faster and, more securely based on open source frameworks.
```
Comment (deadlight) [inline templates/engage/mir-for-iot.html]:
```suggestion
        Industrial robots, home appliances, advertising screens, office information boards&helip; devices of every type around us are getting connected. As they do, their screens turn from single purpose displays to reconfigurable, multi-purpose smart display. As the amount of code required to build these displays, the production time and maintenance burden have increased, this has prompted device manufacturers to reconsider how they can build smart display devices faster and, more securely based on open source frameworks.
```

File: templates/engage/mir-for-iot.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
+{% extends_with_args "engage/base_engage.html" with title="Build smart display devices with Mir Whitepaper" meta_image="https://assets.ubuntu.com/v1/8b2a5baf-61620202-4f636100-ac68-11e9-9c88-ffdccb58ecdf.jpg" meta_description="Mir is an open-source library that simplifies the creation of graphical interfaces for secure, smart display devices." %}
+
+{% block meta_copydoc %}https://docs.google.com/document/d/11E9O_mVN_U63ZLsZOvwsdXAyNtpRv0aF8_VWBDfrnRc{% endblock meta_copydoc %}
+
+{% block content %}
+
+{% include "engage/shared/_header.html" with title="Build smart display devices with Mir: fast to production, secure, open-source" url="#the-form" cta="Download the whitepaper" class="p-strip--dark p-takeover--mir"%}
```
Comment (deadlight) [inline templates/engage/mir-for-iot.html]:
```suggestion
{% include "engage/shared/_header.html" with title="Build smart display devices with Mir: fast to production, secure, open-source" url="#the-form" cta="Download the whitepaper" class="p-strip--dark p-takeover--mir" %}
```
Just adding a space for tidier code

**Comment by anasereijo** [comment]
👍

**Comment by anthonydillon** [comment]
@pmahnke is this still required?

**Comment by pmahnke** [comment]
yes, kill it.

**Pull request closed without merging**